01 Operator Overloading/OperatorOverloading.Model/Money.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OperatorOverloading.Model
+{
+    public class Money
+    {
+        public double Amount { get; set; }
+        public string Currency { get; set; }
+
+        //The operator 
+        public static Money operator+(Money money1,Money money2)
+        {
+         
+            if (!string.IsNullOrEmpty(money1.Currency) && !string.IsNullOrEmpty(money2.Currency) && string.Equals(money1.Currency.ToUpper(),money2.Currency.ToUpper()))
```

Comment:
What will happen if either of parameter is nul? Explore `string.Equals`, you don't need to do `.ToUpper`